### PR TITLE
Add 3rd person viewport support

### PR DESCRIPTION
**Pull request by gnavvy**
First attempt on adding the 3rd person viewport support.
Now testing in the examples/wip/first-person-map (I know, will add a separate example later)

screen shot 2017-10-13 at 5 54 31 pm

### CODE REVIEW

File: examples/wip/first-person-map/app.js [hunk 1]
```diff
@@ -22,8 +23,10 @@ import {StaticMap} from 'react-map-gl';
 
 // Source data CSV
 const DATA_URL = {
-  BUILDINGS: 'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/buildings.json',  // eslint-disable-line
-  TRIPS: 'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/trips.json'  // eslint-disable-line
+  BUILDINGS:
+    'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/buildings.json', // eslint-disable-line
```
Comment (ibgreen) [inline examples/wip/first-person-map/app.js]: Why this change? Should we drop the `eslint-disable-line` since this was only intended to allow us to break the length limit? If we are reformatting to avoid breaking line max, then drop.

File: examples/wip/first-person-map/app.js [hunk 2]
```diff
@@ -167,7 +176,7 @@ class Root extends Component {
         id: 'trips',
         data: trips,
         getPath: d => d.segments,
-        getColor: d => d.vendor === 0 ? [253, 128, 93] : [23, 184, 190],
+        getColor: d => (d.vendor === 0 ? [253, 128, 93] : [23, 184, 190]),
```
Comment (ibgreen) [inline examples/wip/first-person-map/app.js]: Why the parenthesis?

File: examples/wip/first-person-map/app.js [hunk 3]
```diff
@@ -187,23 +196,27 @@ class Root extends Component {
...
         radiusPixels: 20
       }),
       new PointCloudLayer({
         id: 'ref-point',
-        data: [{
-          position: [-1, 0, 2],
-          color: [255, 0, 0, 255],
-          normal: [1, 0, 0]
-        }],
+        data: [
```
Comment (ibgreen) [inline examples/wip/first-person-map/app.js]: I guess this could be a constant.

File: examples/wip/first-person-map/app.js (hunk 2 above)
Comment (gnavvy) [reply examples/wip/first-person-map/app.js]: just saw these comments, those are from the prettier plugins with no functional changes.
I'll disable prettier for deck.gl before we decide to adopt it.
